vernac/comAssumption.mli

```diff
@@ -56,7 +56,14 @@ val declare_axiom
 
 (** Context command *)
 
-val context
+val do_context
   :  poly:bool
   -> local_binder_expr list
   -> unit
+
+(** The first half of the context command, from expr to constr *)
+val interp_context
```

Comment:
I have a bit of confusion with terminology here, so usally we call interp to the phase constexpr to glob_constr, however here we are also calling pretyping. The sigma also play a role, but seems to be needed by accident, so maybe we can put a comment "cleanup sigma uses in do_context"?

Reply:
Regarding the terminology I'm fine with whatever we choose, but IMHO we should use something different from interp for what this is which is interp+pretype.

Reply:
>usally we call interp to the phase constexpr to glob_constr

No, that's intern. see eg Constrintern.intern_constr and Constrintern.interp_constr

Reply:
Yes, I get confused by the directory name being `interp`, interp is the compostion of intern + pretyping.

Maybe we could indeed think about the terminology.